surveys/2022-annual-survey/questions.md

```diff
@@ -364,60 +328,96 @@ Activities:
 
 Choices:
 
-- I've *often* felt welcome
-- I *sometimes* feel welcome
-- I *never* feel welcome
+- I feel welcome
+- I feel neutral
+- I feel unwelcome
 - I've never participated in this activity
 
 > **justification**
 >
 > We'd like to know where people are feeling welcome and the degree to which they are feeling welcome.
 
-### How often have you felt *un*welcome in the Rust community?
 
-Type: matrix (optional)
+### If you indicated that you did not feel welcome in the Rust community, are there any details about your experience that you would like to share with us?
 
-Activities:
+Type: free form (optional)
 
-- *Official* Rust community forums or chats (users.rust-lang.org, internals.rust-lang.org, the official Rust Discord, or the Rust Zulip)
-- *Unofficial* Rust community forums or chats (e.g., reddit.com/r/rust, Hacker News, the Rust *Community* Discord, etc.)
-- Attending a Rust conference
-- Attending a Rust meetup or local community event
-- Discussion (issues, pull requests, etc.) on a repository *inside* the rust-lang GitHub organization
-- Discussion (issues, pull requests, etc.) on a repository *outside* of the rust-lang GitHub organization
+> **justification**
+> 
+> More detail on the type of situations where people have felt unwelcome can let us better 
+> address these issues in the future.
 
-Choices:
+## Rust at work
 
-- I've *often* felt unwelcome
-- I *sometimes* feel unwelcome
-- I *never* feel unwelcome
-- I've never participated in this activity
+### Are you personally using Rust at work?
+
+Type: select one
+
+- Yes, for the majority of my coding
+- Yes, it's one of a number of languages I use and I use it regularly
+- Yes, but I only use it occasionally
+- No (skip to `To what extent is Rust currently being used by your company?`)
 
 > **justification**
 >
-> We'd like to know where people are feeling unwelcome and the degree to which they are feeling unwelcome. This can 
-> help us better understand the free from responses that will come in the next question.
+> We want to establish what percentage of those who could possibly use Rust in a professional setting
+> are using Rust in a professional setting. This is most interesting over time.
+> Answers to this question should be combined with whether the respondent has ever used Rust.
+
+
+### Which of the following statements are reasons why your team uses Rust at work
```

Comment:
We ask about people "personally" using Rust at work, and then jump to their "team". Not sure if that's a problem though.

Reply:
Perhaps just saying "why you and your team"?